### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by edumco**
## Proposed Changes

- Moves the subtitle to another line
- Changes the headers to follow title order 

```markdown
# Main title
## Section
### Subsection
```

# Main title

### CODE REVIEW

**Comment by edumco** [comment]
@googlebot I signed it!

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# Kapitan: Generic templated configuration management for Kubernetes, Terraform and other things
+# Kapitan <img src="./docs/images/kapitan_logo.png" width="250" align="right">
```
Comment (adrianchifor) [inline README.md:1]: Img doesn't look great with right align, please leave it like before :)

File: README.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
...
+
+Generic templated configuration management for Kubernetes, Terraform and other things
```
Comment (adrianchifor) [inline README.md:3]: dot at the end

File: README.md (hunk 1 above)
Comment (edumco) [reply README.md]: Done!

**Pull request merged**